riak_repl/README.md

```diff
@@ -0,0 +1,81 @@
+# Agent Check: Riak-Repl
+
+## Overview
+
+This check monitors Riak replication [riak-repl][1].
+
+## Setup
+
+### Installation
+
+The riak-repl check is not included in the [Datadog Agent][2] package, so you will need to install it yourself.
+
+### Configuration
+
+1. Edit the `riak_repl.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your riak_repl performance data. See the [sample riak_repl.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4]
+
+### Validation
+
+[Run the Agent's `status` subcommand][5] and look for `riak_repl` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
```

Comment:
The metrics below are pulled from `riak_repl/metadata.csv` when the documentation site builds. These should be deleted and replaced with the line:
See [metadata.csv][7] for a list of metrics provided by this integration.

Add this link to the bottom:
[7]: https://github.com/DataDog/integrations-core/blob/master/riak_repl/metadata.csv

Reply:
I believe I implemented your changes correctly... 🤞